pymoveit2/moveit2.py

```diff
@@ -872,6 +894,52 @@ def _plan_kinematic_path(
                 f"Planning failed! Error code: {res.error_code.val}."
             )
             return None
+    def _plan_cartesian_path(
+        self, wait_for_server_timeout_sec: Optional[float] = 1.0
+    ) -> Optional[JointTrajectory]:
+
+        # Re-use request from move action goal
+        self.__cartesian_path_request.start_state = self.__move_action_goal.request.start_state
+        self.__cartesian_path_request.group_name = self.__move_action_goal.request.group_name
+        self.__cartesian_path_request.link_name = self.__end_effector_name
+        self.__cartesian_path_request.max_step = 0.0025
+
+        stamp = self._node.get_clock().now().to_msg()
+        self.__cartesian_path_request.header.stamp = stamp
+
+        self.__cartesian_path_request.path_constraints = self.__move_action_goal.request.path_constraints
+        for position_constraint in self.__cartesian_path_request.path_constraints.position_constraints:
+            position_constraint.header.stamp = stamp
+        for orientation_constraint in self.__cartesian_path_request.path_constraints.orientation_constraints:
+            orientation_constraint.header.stamp = stamp
+        #for joint_constraint in self.__cartesian_path_request.path_constraints.joint_constraints:
+            #joint_constraint.header.stamp = stamp
+
+        target_pose = Pose()
+        target_pose.position = self.__move_action_goal.request.goal_constraints[-1].position_constraints[-1].constraint_region.primitive_poses[0].position
+        target_pose.orientation = self.__move_action_goal.request.goal_constraints[-1].orientation_constraints[-1].orientation
+
+        self.__cartesian_path_request.waypoints = [target_pose]
```

Comment:
Planning Cartesian paths might require a separate interface in order to support multi-waypoint trajectories.
I can just create an issue for that right now, no need to address it here.

Reply:
I see. I have created https://github.com/AndrejOrsula/pymoveit2/issues/10 just to keep track of it (makes it easier for other people to find missing features).